refactor(*) integrate with `aws-lambda`

murillopaula opened the pull request:
This commit integrates the `liamp` plugin with the `aws-lambda` plugin.

The `liamp` plugin should be renamed back to `aws-lambda`, stay in its
own external repo, and then the upstream changes would be to remove
Lambda there, and update the rockspec to include this external one.

Commit:
refactor(*) integrate with `aws-lambda`

This commit integrates the `liamp` plugin with the `aws-lambda` plugin.

The `liamp` plugin should be renamed back to `aws-lambda`, stay in its
own external repo, and then the upstream changes would be to remove
Lambda there, and update the rockspec to include this external one.

Tieske commented on kong-plugin-aws-lambda-0.2.0-1.rockspec:
```diff
@@ -11,7 +11,7 @@ local pluginName = package:match("^kong%-plugin%-(.+)$")  -- "myPlugin"
 supported_platforms = {"linux", "macosx"}
 source = {
-  url = "http://github.com/Tieske/kong-plugin-liamp.git",
+  url = "http://github.com/Tieske/kong-plugin-aws-lambda.git",
```
Tieske -> kong

Tieske reviewed (changes requested):
repo name needs to be updated (see comment), also the version needs to be bumped to 3.0.0 to supersede the current 2.0.0 in Kong open source

Tieske commented:
note: version bump should also reflect in the rockspec filename and contents

murillopaula commented:
Looking at the build logs [1] Kong seems to prioritize `aws-lambda` from the core repository instead of this custom one. So In order for the tests to pass I believe we'll need to remove `aws-lambda` from core first.

[1] https://travis-ci.com/Kong/kong-plugin-liamp/jobs/240845318#L728-L729

Commit:
chore(*) bump plugin version to 3.0.0

We're bumping from 0.2.0 to 3.0.0 to supersede the `aws-lambda` plugin
in the Kong/kong open source repository.

Tieske committed:
chore(rockspec) remove boilerplate code/comments
